src/index.js

```diff
@@ -214,12 +230,35 @@ export default class RNPickerSelect extends PureComponent {
         return {};
     }
 
+    async measureTextInput() {
+        return new Promise((resolve) => {
+            this.inputRef.measureInWindow((_x, y, _width, height) => {
+                resolve({ y, height });
+            });
+        });
+    }
+
+    async handleScrollOnOpenClose() {
+        if (this.props.scrollViewRef != null && this.inputRef != null) {
+            const textInputMeasurements = await this.measureTextInput();
+            const textInputBottomY = textInputMeasurements.y + textInputMeasurements.height;
+            const modalY = Dimensions.get('window').height - IOS_MODAL_HEIGHT;
+
+            if (textInputBottomY > modalY) {
```

Comment:
It would be nice to add some code comments to explain why this logic is here and what it's intention is.